docs/api/taxons.rst

```diff
@@ -931,3 +931,62 @@ Exemplary Response
 .. code-block:: text
 
     STATUS: 204 No Content
+
+Set position of product in a Taxon
+----------------------------------
+
+The products in Sylius can by group by taxon, therefore every product has an relation between itself and selected taxon.
```

Comment:
`The products in Sylius can by grouped by taxon, therefore for every product there is a relation between the product and the assigned taxon.`